crates/wasmcloud-host/src/control_interface/handlers.rs

```diff
@@ -277,7 +277,7 @@ pub(crate) async fn handle_start_actor(host: &str, msg: &nats::asynk::Message, a
         }
     }
 
-    let bytes = crate::oci::fetch_oci_bytes(&cmd.actor_ref, allow_latest).await;
+    let bytes = crate::oci::fetch_oci_bytes(&cmd.actor_ref, allow_latest, false).await;
```

Comment:
Same comment as above, you can see how the `allow_latest` value here is from the `ControlOptions` struct